### ISSUE TITLE
Vanish and Shadowmeld reset combat.

### ISSUE THREAD

**Issue by Bajeru** [dalaranwow/dalaran-wow#5023]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
When you shadowmeld or use stealth in combat (bossfights etc..) it can reset your combat status making you out of combat. Vanish is 100% secure on removing combat, where shadowmeld will work if you are a bit away from the encounter. That can allow rogues with their vanish, to reset the cooldown on their consumables like potions, giving them advantage using multiple potions during the encounter to gain unfair advantage. With shadowmeld, very common at healers to get an extra combat resseruction or drink mana/health restoring food, this will also gain unfair advantage.
**Expected behaviour**: Tell us what should happen instead
Vanish should only cause stealth, and remove movement imparing effects. And shadowmeld is only intended to make enemys less likely to attack you by stealthing in place.
**Steps to reproduce the problem**:

1. Engane encounter
2. Use Vanish or Shadowmeld
3. Win win for rogues and nightelves

**Include proofs for this behaviour**
https://www.youtube.com/watch?v=w-1k3ruqnKI&feature=youtu.be

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
https://wowgaming.altervista.org/aowow/?spell=26889
https://wowgaming.altervista.org/aowow/?spell=62196

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Bajeru** [comment]
Hello alexuktc.

Indeed they would use multiple haste pots, but that number is two. It wasnt possible to vanish in retail, and reset the cooldown on your potions. And the only way it should remove combat in a boss encounter, same story for shadowmeld, is if you are the last party/raid member alive in that encounter, and using your ability.

**Comment by alexuktc** [comment]
Hm, read up on it and it was indeed changed in 3.0.9 . It actually makes it easier for me since pots are extremely expensive anyway.

Report is correct.

**Comment by TeridanDW** [comment]
Hello, after few tests on PTR, both shadowmelt and vanish still reset the CD of the potions during encounter.
![image](https://user-images.githubusercontent.com/6797611/32701807-c62b8a5c-c7e4-11e7-9451-2912faf77929.png)

**Comment by Bajeru** [comment]
Fix soon?

**Comment by Chenocide** [comment]
As far as I know, this should be the case for Feign Death too, does these changes fix all 3 abilities or just Shadowmeld and Vanish?

**Comment by mercred** [comment]
Also whatever the fix is I hope it doesn't break the Tear Gas vanish mechanic on Professor Putricide. Could you please comment on that as well?
